src/settings/toml/target.rs

```diff
@@ -9,7 +9,7 @@ use std::path::PathBuf;
 
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
```

Comment:
what do we use `Default` for? what is a `Default` target?

Reply:
this snuck in while i was doing some work with prototyping on  the deployments project. all a default  here would be is each field would be set to its type's default -- empty `String`, `None`  for `Option`, etc. deriving this here is just a convenience, i can leave it or not.